tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CosmosTests.cs

```diff
@@ -66,7 +66,7 @@ public async Task SubmitTraces(string packageVersion, string metadataSchemaVersi
 
                 var settings = VerifyHelper.GetSpanVerifierSettings();
                 await VerifyHelper.VerifySpans(spans, settings)
-                                  .UseTextForParameters($"Schema{metadataSchemaVersion.ToUpper()}")
+                                  .UseTextForParameters($"Schema{metadataSchemaVersion.ToUpper()}_v{packageVersion}")
```

Comment:
I believe we want to avoid the version specifyer as we are supposed to get the same data across version. Also, by checking the snapshots locally, they are identical, so I believe we shouldn't do this change